cea/demand/thermal_loads.py

```diff
@@ -974,10 +974,7 @@ def calc_prop_rc_model(self, occupancy, architecture, thermal_properties, geomet
         df['Af'] = df['GFA_m2'] * df['Hs']  # conditioned area - areas not heated
         df['Aef'] = df['GFA_m2'] * df['Es']  # conditioned area only those for electricity
 
-        # FIXME: why are we hard-coding 'Cm' here? and can we do without it?
-        if 'Cm' in self.get_overrides_columns():
-            df['Cm'] = self._overrides['Cm']
-        else:
+        if 'Cm' not in self.get_overrides_columns():
```

Comment:
i reverted this change in a new commit. However i see there might be a problem in calling get_overrrides_columns(). the function will overwrite always teh outputs as long as the file overwrite.csv exists. but what happens if i do not erase this file and i wanna use it a normal mode? is this file created and erased after the sensitivity anaysis is complete? also can we change the location of overwrites.csv to the output/data/sensitivity? @daren-thomas